README.md

```diff
@@ -412,7 +454,7 @@
 
       anonymous(); // => TypeError anonymous is not a function
 
-      var anonymous = function () {
+      var anonymous = function() {
```

Comment:
Oop! Sorry, just noticed you missed one here... Should be `var anonymous = function () {`